README.md

```diff
@@ -71,6 +72,9 @@ ghup content -b feature-branch -u local/file.txt:remote/path.txt
 # Create a pull request with changes
 ghup content -b new-feature -u config.json --pr-title "Update configuration"
 
+# Create a pull request with auto-merge enabled (if repository supports it)
```

Comment:
I love the intent here, but I think we should probably introduce this alongside the ability to specify the desired merge method (e.g. merge vs. squash) by exposing the [mutation's mergeMethod field](https://docs.github.com/en/graphql/reference/input-objects#enablepullrequestautomergeinput). Perhaps rename the `--auto-merge` flag to `--pr-auto-merge` (to fit the other PR-related flags) and change the type from a boolean toggle to a [choiceflag](pkg/choiceflag) with options `off` (default), `merge`, `squash` and `rebase`?

Reply:
done